tests/modelfree/test_experiments.py

```diff
@@ -53,6 +56,13 @@ def test_score_agent(config):
     outcomes = [run.result[k] for k in ['ties', 'win0', 'win1']]
     assert sum(outcomes) == run.config['episodes']
 
+    if 'record_traj' in config:
```

Comment:
Change to `if config.get('record_traj', False)`

We don't want this to apply if `'record_traj': False` is in the config.